app/views/admin/users/actions/_banish.html.erb

```diff
@@ -3,14 +3,14 @@
     <% if @user.comments.where("created_at < ?", 100.days.ago).empty? && @user.created_at < 100.days.ago %>
       <p><strong>This is extremely destructive.</strong> Banishing will delete all the user's existing content and change their username to <em>@spam_###</em>.</p>
       <p><strong>Do not do this lightly.</strong></p>
-      <%= form_for(@user, url: banish_admin_user_path(@user), html: { method: :post, onsubmit: "return confirm('Are you sure? This is extremely destructive and irreversible. Banishing will delete all articles and turn their username into @spam_###')" }) do %>
+      <%= form_for(@user, url: banish_admin_user_path(@user), html: { method: :post, onsubmit: "return confirm('Are you sure? This is extremely destructive and irreversible. Banishing will delete all articles and turn their username into @spam_###')", id: nil }) do %>
```

Comment:
From what I found on the internet, the best way seems to be to set `:id => nil` to unset the `id` attribute that gets set by the `form_for` Rails helper.

Happy to change this if there is a more Rails centric way of doing this.